scripts/zones/Tavnazian_Safehold/npcs/Yurim.lua

```diff
@@ -119,10 +119,11 @@ function onEventFinish(player,csid,option)
         player:confirmTrade()
     elseif csid == 529 then
         local itemInProgress = player:getVar("NAME_OF_SCIENCE_target")
```

Comment:
This one too? Should not need to persist through zonage

Reply:
That one must remain getVar.  It's a two-step process: trade the base item + chip, then (optionally) trade an apple pie, then trade the organs.  Player could logout after the first step, so a record of the thingee they're upgrading should persist.

Reply:
oh ok.

Reply:
the apple pie thing is new to me, didn't catch that